chapters/remote/deploy-jinad-on-aws-with-terraform.md

```diff
@@ -0,0 +1,125 @@
+# Deploying JinaD on AWS With Terraform
+
+## Overview
+
+Here is a [terraform module jinad-aws](https://registry.terraform.io/modules/jina-ai/jinad-aws/jina/latest) to deploy `JinaD` on AWS by terraform.
+
+With this terraform module you can:
+- Custom the type and size of the root disk
```

Comment:
Customise the type and size of the root disk

Reply:
Done.